ITunesSearch/ITunesSearch/SearchResultController.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  SearchResultController.swift
+//  ITunesSearch
+//
+//  Created by Taylor Lyles on 5/14/19.
+//  Copyright © 2019 Taylor Lyles. All rights reserved.
+//
+
+import Foundation
+
+
+class SearchResultController {
+	let baseURL = URL(string: "https://itunes.apple.com/search")!
+	var searchResults: [SearchResult] = []
+	
+	func performSearch(searchTerm: String, resultType: ResultType, completion: @escaping (Error?) -> Void) {
+		
+		let searchURL = baseURL.appendingPathComponent("searchResults")
+		
```

Comment:
Then here I would add the baseUrl

`var urlComponents = URLComponents(url: baseURL, resolvingAgainstBaseURL: true)`